app/login/page.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+
+import React from "react";
+import { SignInButton, SignOutButton } from "@/components/buttons";
+
+const LoginPage: React.FC = () => {
+  return (
+    <main className="flex min-h-screen items-center justify-around">
+      <SignInButton />
+      <SignOutButton />
```

Comment:
Same as before. Should be in the menu

Reply:
I agree. However until we have the appbar up and working correctly these does the job smoothly